config/environments/development.rb

```diff
@@ -51,4 +51,6 @@
 
   Slim::Engine.set_options pretty: true, sort_attrs: false
 
+  # Sends email notifications, viewable on the server log
+  config.action_mailer.default_url_options = { :host => 'localhost' }
```

Comment:
This just works on development. Is there something we need to do for production?